.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -48,6 +48,8 @@ stages:
                 echo "install Cilium onto Overlay Cluster"
                 kubectl apply -f test/integration/manifests/cilium/cilium-agent
                 kubectl apply -f test/integration/manifests/cilium/cilium-operator
+                envsubst '${CILIUM_VERSION_TAG}' < test/integration/manifests/cilium/daemonset.yaml | kubectl apply -f -

```

Comment:
Is there a default value for this tag Or we need to provide for each run ?

Reply:
I currently have it set within each pipelines user defined variables. When a update occurs it we will have to change it in 3 locations ( 1 for each pipeline that is currently using the E2E-step-template for cilium). 

The other way of doing this is to add parameters to the template call in order to pass the variable. Which still results in having to change it per pipeline.

Reply:
Yeah but in the pipeline, we don't need a PR so would be faster than updating the file then PR.

Either way, we need to add doc so that people are aware of these variables.

Reply:
who sets CILIUM_VERSION_TAG in nightly pipeline?

Reply:
All pipelines have this tag set within their user defined variables, as such it will stay the same throughout unless changed by us.